AsyncDisplayKitTests/ASVideoNodeTests.m

```diff
@@ -21,6 +21,7 @@ @interface ASVideoNodeTests : XCTestCase
 
 @interface ASVideoNode () {
```

Comment:
In the category () probably put Tests or something if these vars should never be used otherwise